_posts/2016-04-22-welcome-to-beyond-scheme-blog.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Welcome to Beyond Scheme blog!"
+date:   2016-04-22 16:07:07 +0100
+author: "Artur Trzop"
+categories: news
+tags: news ruby java elixir
```

Comment:
tags are visible as blue bubbles under post title. They are also shared as hashtags when you share post on twitter